Source/WebCore/html/canvas/WebGLRenderingContextBase.cpp

```diff
@@ -3964,7 +3970,7 @@ void WebGLRenderingContextBase::texImageImpl(TexImageFunctionID functionID, GCGL
         pixels = std::span<const uint8_t> { data.data(), data.size() };
     }
 
-    ScopedUnpackParametersResetRestore temporaryResetUnpack(this, true);
+    ScopedTightUnpackParameters temporaryResetUnpack(*this, true);
```

Comment:
Previously (Line 3668) this is written as `ScopedTightUnpackParameters temporaryResetUnpack(*this)`. Remove the `true` or the default parameter to `ScopedTightUnpackParameters` and be explicit everywhere.